### ISSUE TITLE
LayerNorm == torch.nn.InstanceNorm1d ?

### ISSUE THREAD

**Issue by hbredin** [mravanelli/SincNet#5]
I believe what you call `LayerNorm` is actually `InstanceNorm1d` in pytorch.
 
https://github.com/mravanelli/SincNet/blob/488c9822e94b1bbfdbf25ba2ca5986632f952909/dnn_models.py#L112-L123

It is my understanding that Layer Normalization would actually have one weight/bias per sample in the sequence, while Instance Normalization only has one per channel.  Do you confirm? 

### References
* Layer Normalization : https://arxiv.org/abs/1607.06450
* Instance Normalization : https://arxiv.org/abs/1607.08022

**Comment by hbredin** [comment]
OK. Thanks.
